### PR TITLE
Documents: Fix brockers status checks #2207 #2273

### LINKED ISSUE #1028: Documentation Bug: As a reader of documentation I'm expecting to see proper configuration options for the management port of Zeebe Gateway and Zeebe Broker
The problem with [this page](https://docs.camunda.io/docs/self-managed/zeebe-deployment/operations/health/), it reflects removed configuration settings about management ports. Also, see: #861

### LINKED ISSUE #2273: Springboot server.host configuration no longer exists
Springboot server.host configuration no longer exists, and was replaced with server.address.

This [update guide](https://github.com/camunda/camunda-platform-docs/blob/cc3e89d45bd5f2b9a5e40b445df83ab3d03bd610/docs/guides/update-guide/120-to-130.md?plain=1#L43-L47) document should reflect that.

### PR DESCRIPTION
**Pull request by rodrigo-lourenco-lopes**
## Description

In this [PR](https://github.com/camunda/camunda-platform-docs/pull/2207) removed the broker status endpoint documentation, when in fact these still exist. This PR reverts these changes.

The configuration values were corrected with the correct configuration name.

Closes #1028 
Closes #2273

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [x] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by akeller — COMMENTED** [review]
Are these endpoints correct? Based on the conversation in the other PR it sounds like this is available in both SaaS and SM.

**Comment by rodrigo-lourenco-lopes** [comment]
@akeller That is a good point. I replace `server.port` with zeebe-broker to be more generic since it can be either `server.port` or `server.host`, and also changed the sentence at the end on how to configure them both.
wdyt?

**Comment by rodrigo-lourenco-lopes** [comment]
While testing the configurability of the server.host, we noticed that this configuration name had been replaced by server.address. An [issue](https://github.com/camunda/camunda-platform-docs/issues/2273) was opened and changes to the update-guide were added in order to reflect this.

File: versioned_docs/version-8.0/self-managed/zeebe-deployment/operations/health.md [hunk 1]
```diff
@@ -4,6 +4,62 @@ title: "Health status"
...
+
+Zeebe broker exposes three HTTP endpoints to query its health status:
+
+- Startup check
+- Ready check
+- Health check
+
+### Startup check
+
+Startup check endpoint is exposed via `http://{zeebe-broker}/startup` (by default port 9600).
```
Comment (megglos) [inline versioned_docs/version-8.0/self-managed/zeebe-deployment/operations/health.md:17]:
```suggestion
Startup check endpoint is exposed via `http://{zeebe-broker-host}:9600/startup`.
The server port is configurable via spring's [server.port](https://docs.spring.io/spring-boot/docs/current/reference/html/application-properties.html#application-properties.server.server.port) property.
```

File: versioned_docs/version-8.1/self-managed/zeebe-deployment/operations/health.md [hunk 2]
```diff
@@ -4,6 +4,62 @@ title: "Health status"
...
+
+Zeebe broker exposes three HTTP endpoints to query its health status:
+
+- Startup check
+- Ready check
+- Health check
+
+### Startup check
+
+Startup check endpoint is exposed via `http://{zeebe-broker}/startup` (by default port 9600).
```
Comment (megglos) [inline versioned_docs/version-8.1/self-managed/zeebe-deployment/operations/health.md:17]:
```suggestion
Startup check endpoint is exposed via `http://{zeebe-broker-host}:9600/startup`.
The server port is configurable via spring's [server.port](https://docs.spring.io/spring-boot/docs/current/reference/html/application-properties.html#application-properties.server.server.port) property.
```

File: versioned_docs/version-8.2/self-managed/zeebe-deployment/operations/health.md [hunk 3]
```diff
@@ -4,6 +4,62 @@ title: "Health status"
...
+
+Zeebe broker exposes three HTTP endpoints to query its health status:
+
+- Startup check
+- Ready check
+- Health check
+
+### Startup check
+
+Startup check endpoint is exposed via `http://{zeebe-broker}/startup` (by default port 9600).
```
Comment (megglos) [inline versioned_docs/version-8.2/self-managed/zeebe-deployment/operations/health.md:17]:
```suggestion
Startup check endpoint is exposed via `http://{zeebe-broker-host}:9600/startup`.
The server port is configurable via spring's [server.port](https://docs.spring.io/spring-boot/docs/current/reference/html/application-properties.html#application-properties.server.server.port) property.
```

**Review by megglos — CHANGES_REQUESTED** [review]
As the port is not a default http port let's add it to the endpoint url listed. To simplify we can also just set it to 9600 and rather hint on how to configure it separately, wdyt?

File: docs/self-managed/zeebe-deployment/operations/health.md [hunk 4]
```diff
@@ -4,6 +4,62 @@ title: "Health status"
...
+
+Zeebe broker exposes three HTTP endpoints to query its health status:
+
+- Startup check
+- Ready check
+- Health check
+
+### Startup check
+
+Startup check endpoint is exposed via `http://{zeebe-broker}/startup` (by default port 9600).
```
[the remark this answers is not in the document]
Comment (rodrigo-lourenco-lopes) [reply docs/self-managed/zeebe-deployment/operations/health.md]: We already have this hint:

`(The default broker port can be configured using environment variables - respectively `SERVER_PORT` and `SERVER_ADDRESS` - or system properties - respectively `-Dserver.port=` or `-Dserver.address=` - to configure them)`

Should we add something else?

[the remark this answers is not in the document]
Comment (megglos) [reply docs/self-managed/zeebe-deployment/operations/health.md]: We can also reuse that here 👍

**Review by megglos — APPROVED** [review]
👍

**Comment by megglos** [comment]
> Are these endpoints correct? Based on the conversation in the other PR it sounds like this is available in both SaaS and SM.

These are actually only relevant for SM, as in SaaS there is no direct access to the broker.

**Comment by rodrigo-lourenco-lopes** [comment]
bors merge

**Comment by megglos** [comment]
> bors merge

no bors here :D Let's also give @akeller the chance to double-check

**Review by akeller — APPROVED** [review]
Great catch on the address pieces @rodrigo-lourenco-lopes! So subtle, but will definitely make users 🤪 .

Thanks for your patience 😄 I will approve and merge this! 🚀

**Pull request merged**